can_tools/scrapers/__init__.py

```diff
@@ -63,6 +63,8 @@
     PennsylvaniaVaccineEthnicity,
     PennsylvaniaVaccineRace,
     PennsylvaniaVaccineSex,
+    SCVaccineCounty,
+    SouthCarolinaTableauVaccineCounty,
```

Comment:
@tlyon3 is this old code now that you are using the pdf?

Reply:
changed